pkg/pool/buffer_pool_test.go

```diff
@@ -1,3 +1,6 @@
+// BufferPool is no-op under race detector, so all these tests do not work.
+// +build !race
+
```

Comment:
Ref: https://github.com/golang/go/blob/master/src/sync/pool_test.go#L5-L6

Reply:
It's a workaround solution, but it will not run unit-testing of this file in current CI flow.

Reply:
Maybe we can do something in `hack/unit-test.sh`, such as adding `-race` dynamically for `go test` :)
https://github.com/dragonflyoss/Dragonfly/blob/12cfd62bfc56554a650c506bc5d51a44fcd115ad/hack/unit-test.sh#L25-L28